qlib/model/trainer.py

```diff
@@ -45,6 +46,46 @@ def begin_task_train(task_config: dict, experiment_name: str, recorder_name: str
     return recorder
 
 
+def fill_placeholder(config: dict, config_extend: dict):
+    """
+    Detect placeholder in config and fill them with config_extend.
+
```

Comment:
Add some documents to tell users that only dict and list are supported. 
The emphasis that tuples are not supported.